compute/compute-node.Dockerfile

```diff
@@ -68,6 +68,8 @@ RUN cd postgres && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/autoinc.control && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/dblink.control && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/postgres_fdw.control && \
+    file=/usr/local/pgsql/share/extension/postgres_fdw--1.0.sql && [ -e $file ] && \
+    echo 'GRANT USAGE ON FOREIGN DATA WRAPPER postgres_fdw TO neon_superuser;' >> $file && \
```

Comment:
I wonder, do we need to grant execute on some helper funcs like `postgres_fdw_disconnect` (some of them are defined in the 1.0 -> 1.1 upgrade script)? Or users can do it without an explicit grant?

Reply:
Users can do it without a grant, I've added this to the private test